src/reactpy_django/templatetags/jinja.py

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2023 Innovatie Ltd. All rights reserved.
+"""
+Jinja support.
+"""
+from django.template import RequestContext, loader
+from jinja2 import pass_context
+from jinja2.ext import Extension
+from jinja2.runtime import Context
+from reactpy_django import config
+from reactpy_django.templatetags.reactpy import component
+
+
+class ReactPyExtension(Extension):
+    """
+    Jinja has more expressive power than core Django's templates, and can
+    directly handle expansions such as:
+
+        {{ component(*args, **kwargs) }}
+    """
+    DJT_TEMPLATE = 'reactpy/component.html'
+    #
+    # Therefore, there is no new tag to parse().
+    #
+    tags = {}
+
+    def __init__(self, environment):
+        super().__init__(environment)
+        #
+        # All we need is to add global "component" to the environment.
+        #
+        environment.globals["component"] = self.template_tag
+
+    @pass_context
+    def template_tag(self, jinja_context: Context, dotted_path: str, *args, host: str | None = None,
+                     prerender: str = str(config.REACTPY_PRERENDER), **kwargs) -> str:
```

Comment:
In order to maintain interface parity more easily, the params for this can be changed to

`(self, jinja_context: Context, *args, **kwargs)`

Reply:
Yes, but then the structural similarity with the native code would be lost, and some annoying/ugly unpacking would be required on line 59. Surely that is worth avoiding?

Reply:
The alternative is I would need to start manually maintaining interface parity between Jinja and Django every time I add a new template tag feature, which is a royal pain.

Reply:
Fair enough. One last thought...since `dotted_path` is required, how would you feel about:
```
def template_tag(self, jinja_context: Context, dotted_path: str, *args, **kwargs)
```

Reply:
Yeah that works.